Formula/pdm.rb

```diff
@@ -18,9 +18,7 @@ class Pdm < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "ddd86f5b824184f535841af9b7249e365c1fb50159100395dcd204f40d688a68"
   end
 
-  depends_on "pygments"
```

Comment:
We can remove `six`, but it looks like `pygments` is still needed. Doing `pip install pdm` inside a clean venv also installs `Pygments`.

Reply:
Good to know, I just reverted this. Will that dependendy be injected into pdm environment?